packaging/cmake/Config.cmake.in

```diff
@@ -7,7 +7,12 @@ if(@WITH_ZLIB@)
 endif()
 
 if(@WITH_ZSTD@)
-  find_dependency(ZSTD)
+  find_library(ZSTD zstd)
```

Comment:
what's the practical difference between find_dependency and find_library?

Reply:
find_dependency(ZSTD) requires a CMake module for ZSTD to be installed on the system. It works similar to find_package(ZSTD). AFAIK there is no CMake module provided by CMake for ZSTD and find_dependency will fail.
I ran into this problem, when I tested to link a program with librdkafka using the CMake module instead of pkg-config:
```
find_package(RdKafka REQUIRED)
add_executable(cmake-test cmake-test.cpp)
target_link_libraries(cmake-test PUBLIC RdKafka::rdkafka++)
```

find_library(ZSTD zstd) on the other hand scans the system directories for libzstd.so or whatever is appropriate for the OS librdkafka is compiled on. This is also the procedure used to detect the ZSTD library in the CMake files of librdkafka (see https://github.com/edenhill/librdkafka/blob/master/CMakeLists.txt#L35)